examples/dnspolicy/application.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: gateway.networking.k8s.io/v1
```

Comment:
is this example unique to dnspolicy?

Reply:
it's an example of the minimum setup that results in DNS Records from a DNS Policy.